### ISSUE TITLE
casi totali incoerenti

### ISSUE THREAD

**Issue by alexodus** [pcm-dpc/COVID-19#1106]
ieri:
    "totale_casi":3376376,
oggi:
    "nuovi_positivi":13846,
    "totale_casi":3400877,

totale_casi dovrebbe essere la somma dei 2 numeri precedenti

```
 {
    "data":"2021-03-21T17:00:00",
    "stato":"ITA",
    "ricoverati_con_sintomi":27484,
    "terapia_intensiva":3448,
    "totale_ospedalizzati":30932,
    "isolamento_domiciliare":540740,
    "totale_positivi":571672,
    "variazione_totale_positivi":6219,
    "nuovi_positivi":20159,
    "dimessi_guariti":2699762,
    "deceduti":104942,
    "casi_da_sospetto_diagnostico":null,
    "casi_da_screening":null,
    "totale_casi":3376376,
    "tamponi":46890903,
    "casi_testati":21873168.0,
    "note":null,
    "ingressi_terapia_intensiva":232.0,
    "note_test":null,
    "note_casi":null,
    "totale_positivi_test_molecolare":3280621.0,
    "totale_positivi_test_antigenico_rapido":95755.0,
    "tamponi_test_molecolare":38553755.0,
    "tamponi_test_antigenico_rapido":8337148.0
  },
  {
    "data":"2021-03-22T17:00:00",
    "stato":"ITA",
    "ricoverati_con_sintomi":28049,
    "terapia_intensiva":3510,
    "totale_ospedalizzati":31559,
    "isolamento_domiciliare":531508,
    "totale_positivi":563067,
    "variazione_totale_positivi":-8605,
    "nuovi_positivi":13846,
    "dimessi_guariti":2732482,
    "deceduti":105328,
    "casi_da_sospetto_diagnostico":null,
    "casi_da_screening":null,
    "totale_casi":3400877,
    "tamponi":47060099,
    "casi_testati":22071115.0,
    "note":null,
    "ingressi_terapia_intensiva":227.0,
    "note_test":null,
    "note_casi":null,
    "totale_positivi_test_molecolare":3304516.0,
    "totale_positivi_test_antigenico_rapido":96361.0,
    "tamponi_test_molecolare":38660491.0,
    "tamponi_test_antigenico_rapido":8399608.0
  }
```

**Comment by Rabelaiss** [comment]
In generale un numero nella serie totale_casi non è la somma tra il corrispondente numero in nuovi_positivi e il numero precedente di totale_casi, questo perché quando avviene un riconteggio il numero precedente in totale_casi non viene corretto, mentre nuovi_positivi tiene conto dei riconteggi.

Oggi
• bolzano ha aggiunto 10665 casi vecchi e ne ha tolti 4
• abruzzo ha tolto 1 caso
• emilia romagna ha tolto 5 casi
Gli ultimi 2 numeri in totale_casi dell'italia sono 3400877 e 3376376, per ottenere il numero di nuovi casi bisogna prima aggiungere 10665 a 3376376 e poi togliere 4, 1 e 5 ottenendo 3387031
quindi i nuovi casi sono
3400877 - 3387031 = 13846

**Comment by alexodus** [comment]
> In generale un numero nella serie totale_casi non è la somma tra il corrispondente numero in nuovi_positivi e il numero precedente di totale_casi, questo perché quando avviene un riconteggio il numero precedente in totale_casi non viene corretto, mentre nuovi_positivi tiene conto dei riconteggi.
> 
> Oggi
> • bolzano ha aggiunto 10665 casi vecchi e ne ha tolti 4
> • abruzzo ha tolto 1 caso
> • emilia romagna ha tolto 5 casi
> Gli ultimi 2 numeri in totale_casi dell'italia sono 3400877 e 3376376, per ottenere il numero di nuovi casi bisogna prima aggiungere 10665 a 3376376 e poi togliere 4, 1 e 5 ottenendo 3387031
> quindi i nuovi casi sono
> 3400877 - 3387031 = 13846

andrebbe ricorretto invece, altrimenti il record precedente non ha più senso

**Comment by Rabelaiss** [comment]
Come si può vedere in [avvisi](https://github.com/pcm-dpc/COVID-19/blob/master/avvisi.md) ogni giorno ci sono piccoli riconteggi e quasi mai vengono comunicate le date a cui fanno riferimento i casi eliminati.
Se infatti confronti i delta di totale_casi con i nuovi_positivi vedrai che quasi mai coincidono, ad esempio negli ultimi 60 giorni solo 3 volte il delta coincide con nuovi_positivi

**Comment by RossoDeM** [comment]
> Come si può vedere in [avvisi](https://github.com/pcm-dpc/COVID-19/blob/master/avvisi.md) ogni giorno ci sono piccoli riconteggi e quasi mai vengono comunicate le date a cui fanno riferimento i casi eliminati.
> Se infatti confronti i delta di totale_casi con i nuovi_positivi vedrai che quasi mai coincidono, ad esempio negli ultimi 60 giorni solo 3 volte il delta coincide con nuovi_positivi

Tutto vero. Ma si continua a non vedere il problema. È proprio il procedimento ad essere sbagliato.

**Comment by andreazinno** [comment]
> > In generale un numero nella serie totale_casi non è la somma tra il corrispondente numero in nuovi_positivi e il numero precedente di totale_casi, questo perché quando avviene un riconteggio il numero precedente in totale_casi non viene corretto, mentre nuovi_positivi tiene conto dei riconteggi.
> > Oggi
> > • bolzano ha aggiunto 10665 casi vecchi e ne ha tolti 4
> > • abruzzo ha tolto 1 caso
> > • emilia romagna ha tolto 5 casi
> > Gli ultimi 2 numeri in totale_casi dell'italia sono 3400877 e 3376376, per ottenere il numero di nuovi casi bisogna prima aggiungere 10665 a 3376376 e poi togliere 4, 1 e 5 ottenendo 3387031
> > quindi i nuovi casi sono
> > 3400877 - 3387031 = 13846
> 
> in teoria andrebbe ricorretto, altrimenti il record precedente non ha più senso. Però in questo caso ha senso visto che non si possono determinare le date in cui "ridistribuire" quei 10mila casi.

Vero, anche se si potrebbero redistribuire i dati sulla base della distribuzione percentuale calcolata sull'intero periodo al quale la rettifica si riferisce. Per fare un esempio: il gran numero di casi aggiunti da Bolzano potrebbe essere redistribuito sull'intervall di tempo nel quale tali dati non son ostati conteggiati, ripartendoli, per ogni giorno in tale intervallo, in base alla percentuale che, in quello stesso giorno, ha caratterizzato il dato già comunicato.

**Comment by Paulsword** [comment]
> > > In generale un numero nella serie totale_casi non è la somma tra il corrispondente numero in nuovi_positivi e il numero precedente di totale_casi, questo perché quando avviene un riconteggio il numero precedente in totale_casi non viene corretto, mentre nuovi_positivi tiene conto dei riconteggi.
> > > Oggi
> > > • bolzano ha aggiunto 10665 casi vecchi e ne ha tolti 4
> > > • abruzzo ha tolto 1 caso
> > > • emilia romagna ha tolto 5 casi
> > > Gli ultimi 2 numeri in totale_casi dell'italia sono 3400877 e 3376376, per ottenere il numero di nuovi casi bisogna prima aggiungere 10665 a 3376376 e poi togliere 4, 1 e 5 ottenendo 3387031
> > > quindi i nuovi casi sono
> > > 3400877 - 3387031 = 13846
> > 
> > 
> > in teoria andrebbe ricorretto, altrimenti il record precedente non ha più senso. Però in questo caso ha senso visto che non si possono determinare le date in cui "ridistribuire" quei 10mila casi.
> 
> Vero, anche se si potrebbero redistribuire i dati sulla base della distribuzione percentuale calcolata sull'intero periodo al quale la rettifica si riferisce. Per fare un esempio: il gran numero di casi aggiunti da Bolzano potrebbe essere redistribuito sull'intervall di tempo nel quale tali dati non son ostati conteggiati, ripartendoli, per ogni giorno in tale intervallo, in base alla percentuale che, in quello stesso giorno, ha caratterizzato il dato già comunicato.

Non è dato sapere nemmeno l'intervallo, purtroppo.
